packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -130,6 +131,10 @@ interface OrgNav {
   admin: boolean;
   hosts: Required<HostsInterface>;
   currentProjectName?: string;
+  isOnPrem?: boolean;
```

Comment:
I wonder if we should encapsulate all onPrem config inside an object.

```jsx
onPrem={{
  enabled: true,
  version: string,
   mfaEnabled: boolean
}}
```

WDYT @brookescarlett @seanmhanson ?

Reply:
I don't hate that... eliminates some props and there are _so_ many on this component

Reply:
Thoughts @seanmhanson?

Reply:
Updated! Let me know what you think!